### DISCUSSION TITLE
How to deal with negative budgets

### DISCUSSION THREAD

**Discussion by TheDoubleJo** [firefly-iii/firefly-iii#4796] [Firefly III]
I'm diving into Firefly coming from a home made 0 based budgeting tool.

By the way, I'm impressed by the overall quality of Firefly, great job!

However, I'm struggling with negative budgets:

I created a rollover budget in which I add 50€ each month.
This is a clothing budget and let's say I managed to get 100€ in it.

Unfortunatly, I need a new jacket now and it costs 150€.

I buy the new jacket, and I link the corresponding transaction to the clothing budget.

This budget is now at -50€.

I accept this negative state since I can wait another month to add another 50€ to finally get to 0€.

I'm still a noob to Firefly but I think it can't handle this use case.

How would you do?

**Comment by JC5** [comment]
Reddit or GitHub take your pick please. Not both.

**Reply by TheDoubleJo** [reply]
Ok sorry.
